test/visibility/import/public/hideHierarchy/MyLibrary.chpl

```diff
@@ -0,0 +1,8 @@
+module MyLibrary {
+  public import Unimportant.Important1;
+  public import Unimportant.Important2;
```

Comment:
Did you end up going back and looking again at the version where Unimportant is also nested?

Reply:
[this commit](https://github.com/chapel-lang/chapel/pull/15192/commits/219e505641476f33ca21485398e156d3689d180d) does that